src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        ball.setColor(colorSupplier.getRandomColor());
+        Random random = new Random();
+        ball.setNumber(random.nextInt(100));
+        return ball;

```

Comment:
1. Please make colorSupplier and random class variables in order not to create new object each time method is called
2. Please make constructor for Ball that accept all required arguments and use it here instead of setters to make code cleaner